src/pages/cocktails.js

```diff
@@ -25,9 +40,22 @@ export default function CocktailPage() {
 			</Head>
 			<>
 				<ul>
```

Comment:
`IngredientGrid` is already a `ul` element. You can remove this `ul` as you don't need it anymore.